dotcom-rendering/package.json

```diff
@@ -21,6 +21,8 @@
 		"test:ci": "jest --runInBand",
 		"createtoc": "doctoc $npm_package_tocList --github --update-only --title '<!-- Automatically created with yarn run createtoc and on push hook -->' ",
 		"addandcommittoc": "git add $npm_package_tocList && git commit -m 'Add TOC update' || true",
+		"playwright:open": "playwright test --ui",
+		"playwright:run": "playwright test",
```

Comment:
would it be useful to add some targets to the `Makefile` for this? I'm guessing there needs to be a server running for playwright to make requests to?

Reply:
Yep good point. I was going add after we delete Cypress as having two ways of running two different e2e tests might be confusing. Happy to add now though.